src/utils/index.ts

```diff
@@ -53,6 +53,77 @@ const promptLayerApiRequest = async (body: TrackRequest) => {
   }
 };
 
+/**
+ * Get a prompt from the PromptLayer library
+ * @param prompt_name name of the prompt to get
+ * @param api_key your api key
+ * @param version version of the prompt to get, None for latest
+ * @param label The release label of a prompt you want to get. Setting this will supercede version
+ */
+const promptLayerGetPrompt = async (body: prompt.Retrieve) => {
+  const params: Record<string, string> = {
+    prompt_name: body.prompt_name,
+    version: body.version?.toString() ?? "",
+    label: body.label ?? "",
+  };
+  const url = new URL(`${URL_API_PROMPTLAYER}/library-get-prompt-template`);
+  url.search = new URLSearchParams(params).toString();
+  let response: Response;
+  try {
+    response = await fetch(url.toString(), {
+      method: "GET",
+      headers: {
+        "X-API-KEY": getApiKey(),
+      },
+    });
+  } catch (e) {
+    throw new Error(
+      `PromptLayer had the following error while getting your prompt: ${e}`
+    );
+  }
+  const data = await response.json();
+  if (response.status !== 200) {
+    throwOnBadResponse(
+      data,
+      `PromptLayer had the following error while getting your prompt `
```

Comment:
should be "retrieving your prompt template" not "getting your prompt

Reply:
Updated.